datasets/hail-vep-pipeline.yml

```diff
@@ -0,0 +1,33 @@
+Name: Variant Effect Predictor (VEP) and the Loss-Of-Function Transcript Effect Estimator (LOFTEE) Plugin
+Description: VEP determines the effect of genetic variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. The European Bioinformatics Institute produces the VEP tool/db and releases updates every 1 - 6 months. The latest release includes The latest release contains 267 genomes from 232 species containing 5567663 protein coding genes. This dataset hosts the last 5 releases for human, rat, and zebrafish. Also, it hosts the required reference files for the Loss-Of-Function Transcript Effect Estimator (LOFTEE) plugin as it is commonly used with VEP.
+Documentation: https://privoit.github.io/hail-vep-pipeline/
+Contact: https://www.privoit.com/contact-us
+ManagedBy: "[Privo](https://www.privoit.com/)"
+UpdateFrequency: New packages are added as soon as they are available and confirmed to work with recent versions of Hail.
+Tags:
+  - genome-wide association study
+  - genomic
+  - life sciences
+  - vep
+  - loftee
+License: [VEP](https://uswest.ensembl.org/info/about/publications.html) use is governed by the Apache 2.0 licenses, and [LOFTEE](https://github.com/konradjk/loftee/blob/master/LICENSE) use is governed by the MIT license.
+Resources:
+  - Description: VEP and LOFTEE data
+    ARN: arn:aws:s3:::hail-vep-pipeline
+    Region: us-east-1
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
```

Comment:
Add Tutorials field (doesn't have to be filled in, but the field has to be there)

Reply:
Added